vowpalwabbit/core/include/vw/core/reductions/epsilon_decay.h

```diff
@@ -57,11 +57,8 @@ class epsilon_decay_data
   bool _predict_only_model;
 
   // TODO: delete all this, gd and cb_adf must respect ft_offset, see header import of automl.cc
-  std::vector<double> per_live_model_state_double;
+  LEARNER::learner* adf_learner = nullptr;  //  re-use print from cb_explore_adf
```

Comment:
Is this change related to the ppw change?

Reply:
removed